CONTRIBUTING.md

```diff
@@ -228,32 +229,40 @@ only pull requests targeted directly against master.
 - Tag the release (use `git tag -a`) and create a release in Github
 - Delete the `RC` branches
 
-### Point Release Procedure
```

Comment:
I would really prefer to keep the nomenclature point releases because that is what they are. In fact, in very rare cases, they include breaking changes.

Reply:
Introducing a breaking change when it's necessary to fix a bug is IMHO not a problem and would be a good rationale to grant an SRU exception. I'll clarify this in the `What qualifies as SRU`